src/core/components/forms/radioGroup/radioGroup/radioGroup.tsx

```diff
@@ -11,14 +32,21 @@ export interface IRadioGroupProps extends Omit<RadioGroupProps, 'orientation'> {
  * An exhaustive list of its properties can be found in the corresponding Radix primitive
  * [documentation](https://www.radix-ui.com/primitives/docs/components/radio-group#root).
  */
-export const RadioGroup = forwardRef<HTMLDivElement, IRadioGroupProps>(({ className, ...rest }, ref) => {
-    return (
-        <PrimitiveRadioGroup
-            ref={ref}
-            className={classNames('flex min-w-0 flex-col gap-y-2 md:gap-y-3', className)}
-            {...rest}
-        />
-    );
-});
+export const RadioGroup = forwardRef<HTMLDivElement, IRadioGroupProps>(
+    ({ className, value, defaultValue, onValueChange, name, disabled, ...rest }, ref) => {
+        return (
```

Comment:
Let's destructure the props object inside the component to avoid adding a tab indentation for the whole component:
```
const Component = (props) => {
  const { ... } = props;
```
Same for the `RadioCard` component

Reply:
✅ https://github.com/aragon/ods/pull/218/commits/1a984b621f19b25f8e817d9c2fcc146d774c659d